client/me/concierge/controller.js

```diff
@@ -9,12 +9,22 @@ import React from 'react';
  * Internal dependencies
  */
 import ConciergeMain from './main';
+import BookCalendarStep from './book/calendar-step';
+import BookConfirmationStep from './book/confirmation-step';
+import BookInfoStep from './book/info-step';
+import BookSkeleton from './book/skeleton';
 
-const concierge = ( context, next ) => {
-	context.primary = React.createElement( ConciergeMain, { siteSlug: context.params.siteSlug } );
+const book = ( context, next ) => {
```

Comment:
Just thinking out loud and I'm sure theres a great reason for renaming this to book but I think it might add a bit of confusion since the components are named "Concierge" and the page route has "concierge" in it. A developer a few months from now searching for something might not be able to easily find what (s)he's looking for.

Reply:
Good point, I moved everything under `book` so we can easily separate different concierge flows. I'll update the url to follow the same pattern as rescheduling and cancel so the book form will be available at `me/concierge/<your-site-slug/book`. That way it will be consistent with other specific section and still have the main shared `Concierge` component.